face-kiosk/face-kiosk-application/src/main/java/com/pillar/skynet/facekiosk/camera/BuiltInCamera.java

```diff
@@ -0,0 +1,26 @@
+package com.pillar.skynet.facekiosk.camera;
+
+import org.bytedeco.javacpp.opencv_core;
+import org.bytedeco.javacv.Frame;
+import org.bytedeco.javacv.FrameGrabber;
+import org.bytedeco.javacv.OpenCVFrameConverter;
+import org.bytedeco.javacv.OpenCVFrameGrabber;
+
+import static org.bytedeco.javacpp.opencv_imgcodecs.cvSaveImage;
+
+public class BuiltInCamera {
+
```

Comment:
Just a note, we could separate taking pictures from saving frames, so the second method could go to another class.